### PR TITLE
Update Rate Limits Information for Dynamic Scale and Workforce Multiplier

### PR DESCRIPTION
**Pull request by christinechoy-okta**
## Description:
- **What's changed?** Add dynamic scale/workforce multiplier info
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-416973](https://oktainc.atlassian.net/browse/OKTA-416973)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md [hunk 1]
```diff
@@ -23,31 +23,31 @@ This page provides the API rate limits for authentication and end-user activitie
...
+| **Authenticate different end users:**<br>`/api/v1/authn`<br>*Eligible for dynamic scale and workforce multiplier*                                                               | 100              | 600              | 600     | 600        | 500                   |
+| **Verify a factor:**<br>`/api/v1/authn/factors/${factorIdOrFactorType}/verify` only<br>*Eligible for dynamic scale and workforce multiplier*                                     | 100              | 600              | 600     | 600        | 500                   |
+| **Get session information:**<br>`/api/v1/sessions`<br>*Eligible for dynamic scale and workforce multiplier*                                                                     | 100              | 600              | 600     | 600        | 750                   |
+| **OAuth2 requests for Custom Authorization Servers:**<br>`/oauth2/${authorizationServerId}/v1` except `/oauth2/${authorizationServerId}/v1/authorize` and public metadata endpoints (see Note below)<br>*Eligible for dynamic scale and workforce multiplier*  | 300 | 1,200 | 1,200     | 1,200       | 2,000                  |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:29]:
```suggestion
| **OAuth2 requests for Custom Authorization Servers:**<br>`/oauth2/${authorizationServerId}/v1` except `/oauth2/${authorizationServerId}/v1/authorize` and public metadata endpoints (see the Note below)<br>*Eligible for dynamic scale and workforce multiplier*  | 300 | 1,200 | 1,200     | 1,200       | 2,000                  |
```

File: packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md [hunk 2]
```diff
@@ -23,31 +23,31 @@ This page provides the API rate limits for authentication and end-user activitie
...
 | `/oauth2/${authorizationServerId}/v1/authorize`                                                                         | 300              | 1200             | 1200    | 1200       | 2000
-| **OAuth2 requests for the Org Authorization Server:**<br>`/oauth2/v1` except `/oauth2/v1/clients`, `/oauth2/v1/authorize` and public metadata endpoints (see Note below) | 300 | 1,200 | 1,200     | 1,200       | 2,000                  |
+| **OAuth2 requests for the Org Authorization Server:**<br>`/oauth2/v1` except `/oauth2/v1/clients`, `/oauth2/v1/authorize` and public metadata endpoints (see Note below)<br>*Eligible for dynamic scale and workforce multiplier* | 300 | 1,200 | 1,200     | 1,200       | 2,000                  |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:31]:
```suggestion
| **OAuth2 requests for the Org Authorization Server:**<br>`/oauth2/v1` except `/oauth2/v1/clients`, `/oauth2/v1/authorize` and public metadata endpoints (see the Note below)<br>*Eligible for dynamic scale and workforce multiplier* | 300 | 1,200 | 1,200     | 1,200       | 2,000                  |
```

File: packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md [hunk 3]
```diff
@@ -23,31 +23,31 @@ This page provides the API rate limits for authentication and end-user activitie
...
-| **Authenticate different end users:**<br>`/api/v1/authn`                                                               | 100              | 600              | 600     | 600        | 500                   |
-| **Verify a factor:**<br>`/api/v1/authn/factors/${factorIdOrFactorType}/verify` only                                     | 100              | 600              | 600     | 600        | 500                   |
-| **Get session information:**<br>`/api/v1/sessions`                                                                     | 100              | 600              | 600     | 600        | 750                   |
-| **OAuth2 requests for Custom Authorization Servers:**<br>`/oauth2/${authorizationServerId}/v1` except `/oauth2/${authorizationServerId}/v1/authorize` and public metadata endpoints (see Note below)  | 300 | 1,200 | 1,200     | 1,200       | 2,000                  |
+| **Authenticate different end users:**<br>`/api/v1/authn`<br>*Eligible for dynamic scale and workforce multiplier*                                                               | 100              | 600              | 600     | 600        | 500                   |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:26]: why the italics? I built locally, and it doesn't look so hot. Can we just have all of the italicized lines be regular text? We avoid the use of italics in the docs. From the style guide:

emphasis | Don't use bold, italics, or underlining for emphasis. If something needs that much attention, consider isolating it as a standalone note or tip.

**Review by susanharper-okta — COMMENTED** [review]
couple nits and switching italicized to regular text.

File: packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md (hunk 3 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:26]: Good point. I'll remove the italics.

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Review by joec-okta — APPROVED** [review]
looks good.

**Pull request merged**